src/components/EnvironmentOverride/DeploymentTemplateOverrideForm.tsx

```diff
@@ -427,11 +358,7 @@ export default function DeploymentTemplateOverrideForm({
             case 1:
             case 3:
```

Comment:
Why are we changing the selection on tab change? Please verify if it is required?